capstone/src/main/java/com/google/step/servlets/LoginServlet.java

```diff
@@ -75,22 +75,29 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
    * @return String containing the words User or Restaurant.
    */
   private static String checkIfRestaurantOrUser(String email) {
+    boolean restaurant = checkUserTypeExists(email, "Restaurant");
+    boolean user = checkUserTypeExists(email, "User");
+    if (restaurant == true && user == false) {
+      return "Restaurant";
+    } else if (restaurant == false && user == true) {
+      return "User";
+    }
+    return "None";
+  }
+
+  /**
+   * Checks if the user type inputted exists.
+   * @param email email of the user
+   * @param type type of user we are looking for
+   * @return returns a boolean stating true if it exists and false if otherwise.
+   */
+  private static boolean checkUserTypeExists(String email, String type) {
```

Comment:
Is there a reason that this method is static?

Reply:
You're right. Changed